### ISSUE TITLE
Better Idea for Non-VR players

### ISSUE THREAD

**Issue by BeigeAnimal** [jrbudda/Vivecraft_Spigot_Extensions#42]
Make it so if Vivecraft Spigot/forge is installed on your server make it so there is an option in the config to force non-vr players to install the non-vr companion version of the mod and I recommend two types of prompts one that the recommends the players install it and one that won't allow the player join the server until they installed.

**Comment by melerpe** [comment]
But that would ultimately limit the amount of players in your server

**Comment by melerpe** [comment]
Mine allows non vr to play nicely with vr

**Comment by BeigeAnimal** [comment]
not if you choose that second option that recommends players to download it

**Comment by BeigeAnimal** [comment]
Duh

**Comment by melerpe** [comment]
Yeah, but with my armorstand sollution, they won't have to download a thing

**Comment by BeigeAnimal** [comment]
Yeah but it would also look worse and probably take more time to do rather than making a prompt that recommends people download the non-vr version mod. I rather see people looking at the ground ,that's generally what you see because a lot of people have their reticle facing the ground when in vr, rather than staring at armor stand plus it could have adverse side effects on their hit boxes plus armor stand don't have any animations to being with so it would look janky af. @melerpe

**Comment by melerpe** [comment]
Armorstand are easely animatable. Hands and head would follow the controllers and head. And you could simulate the walking animation easily. (just plop the players head on the armorstand. And the things it's holding/wearing

**Comment by BeigeAnimal** [comment]
@melerpe if it's so easier why don't you do it? The source code is available to download. Do it as a proof of conception and then request them to add your additions to the main build.

**Comment by melerpe** [comment]
First off all. I will try and create it partially (no leg movement and sneaking support)
Also it will be ineffecient as hell. Because I am not that good with Java and all.

**Comment by melerpe** [comment]
Prob something Like, get controleposition and head position. Use armorstand size to get relative position to armorstand. Calculate the eulerangle based on location of armorstand(prefer the arm start location)  and controller. And apply the eulerangle to the armorstand do that 20 times a second and profit. I will start coding tomorrow orzo because I currently do not have access to a pc

**Comment by MilesBHuff** [comment]
Can do the armorstand option AND this option, by the way.  They aren't mutually exclusive.
